gpu_bdb/bdb_tools/q27_get_review_sentence_utils.py

```diff
@@ -233,6 +233,7 @@ def get_org_sentences(sentence_boundary_df, org_df):
         inplace=True,
     )
     valid_left_loc = valid_left_loc.reset_index(drop=False)
+    valid_left_loc = valid_left_loc.rename(columns={"index":"flat_loc_org"})
```

Comment:
Had to this because we otherwise were failing with no column found in  `valid_df = valid_df.set_index(["flat_loc_org"])`